tests/std/tests/Dev11_0577418_random_seed_0/test.cpp

```diff
@@ -24,7 +24,7 @@ int main() {
     // Also test VSO-214595 "subtract_with_carry_engine::seed should accept result_type"
     subtract_with_carry_engine<unsigned long long, 64, 10, 24> ull_swc;
     ull_swc.seed(0x12341234'00000000ULL);
-    assert(run_10k(ull_swc) == 0x1DD6C263'C41EEED0ULL); // value changed by LWG-3809
+    assert(run_10k(ull_swc) == 0x01316AEA'3646F686ULL); // libstdc++ and libc++ agree (boost 1.60.0 disagrees)

```

Comment:
This is great, but Boost 1.60.0 is incredibly ancient (Dec 2015), so this comment sounds outdated. It remains true for the current version 1.86.0 (see https://godbolt.org/z/e84Ws7Y8c), so let's cite that.

Let's also cite specific versions of libstdc++ and libc++, just in case they change (or have changed) their behavior.